pkg/config/config.go

```diff
@@ -11,18 +12,23 @@ type DaemonConfig struct {
 	PeriodicUpdate int `env:"PERIODIC_UPDATE"` // Interval between every check for the added and deleted pods
 	GuidPool       GuidPoolConfig
 	SubnetManager  SubnetManagerPluginConfig
+	Kube           KubeConfig
 }
 
 type GuidPoolConfig struct {
-	RangeStart string `env:"RANGE_START"` // First guid of the pool
-	RangeEnd   string `env:"RANGE_END"`   // Last of the guid pool
+	RangeStart string `env:"GUID_RANGE_START"` // First guid of the pool
+	RangeEnd   string `env:"GUID_RANGE_END"`   // Last of the guid pool
 }
 
 type SubnetManagerPluginConfig struct {
 	Plugin string `env:"PLUGIN"` // Subnet manager plugin name
 	Ufm    UFMConfig
 }
 
+type KubeConfig struct {
```

Comment:
nit: KubeClientConfig ?

im thinking about the next PR that will Prefix each config attribute with its corresponding struct name
and  `KUBE_CLIENT_KUBECONFIG_FILE` seems clearer than `KUBE_KUBECONFIG_FILE`